app/models/data.py

```diff
@@ -252,6 +252,27 @@ def totalTestResults(self):
     def parse_str_to_date(date_input):
         return parser.parse(str(date_input), ignoretz=True).date()
 
+    @staticmethod
+    def valid_fields_checker(candidates):
+        '''
+        dict[string] -> ([string], [string])
+        Gets a list of field names and returns a tuple of (valid_fields, unknown_fields).
+
+        If valid_fields is empty, then this list contains no data for this object.
+        If unknown_fields is not-empty, then this list contains extra fields
+        that mean nothing for this object, but we might want to alert on this.
+        In the valid fields, we exclude state, date and batchId
```

Comment:
could you add a small note for why we exclude those 3 from the valid fields? it'll be helpful for later when we forget context (not totally obvious)